### PR TITLE
antd theme related changes, bump v4 to v5 

### PR DESCRIPTION
**Pull request by Manish3323**
### Reasons for making this change

[AntD version upgrade](https://github.com/rjsf-team/react-jsonschema-form/issues/3639)

Somehow my fork's main branch got all its commits discarded, that resulted in Automatic closing of the earlier PR.

### CODE REVIEW

File: packages/antd/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "@ant-design/icons": "^4.0.0",
     "@rjsf/core": "^5.12.x",
     "@rjsf/utils": "^5.12.x",
-    "antd": "^4.0.0",
+    "antd": "^4.24.0",
```
Comment (nickgros) [inline packages/antd/package.json:40]: Shouldn't this be `"^4.24.0 || ^5.8.5" ? Otherwise antd v5 users will get a peer dependency warning/error

Comment (Manish3323) [reply packages/antd/package.json:40]: the caret symbol will handle it right?.. npm installs latest version of v4..

Comment (Manish3323) [reply packages/antd/package.json:40]: Ah okay.. Understood..

Comment (Manish3323) [reply packages/antd/package.json]: Fixed

**Pull request merged**